src/main/java/com/google/sps/model/TasksClientImpl.java

```diff
@@ -61,6 +62,17 @@ private TasksClientImpl(Credential credential) {
     return taskLists != null ? taskLists : new ArrayList<>();
   }
 
+  @Override
+  public List<TaskList> listTaskLists(List<String> taskListTitles) throws IOException {
+    // returns null if no taskLists exist. Convert to empty list for ease.
+    List<TaskList> taskLists = tasksService.tasklists().list().execute().getItems();
```

Comment:
Call `listTasklists()` here to avoid code duplication

Reply:
Great point, thanks!

Reply:
`RESOLVED`